### ISSUE TITLE
Setup script

### ISSUE THREAD

**Issue by NegativeK** [NegativeK/picamserver#25]
Create a setup process to handle all of the apt installs, etc., that are necessary. The user should not need to take any further steps to have a working picamserver with good default options.

**Comment by NegativeK** [comment]
Thoughts on implementations: bash script or maybe a Debian dpkg?

This is dependent on #10.

**Comment by mrjones-plip** [comment]
i've not authored a `.dpkg` before, so I'd say KISS with a bash script.  however, maybe `.dpkg` is easier than I think?

**Comment by NegativeK** [comment]
Yeah; bash script would probably wise even if only for the interim.

**Comment by mrjones-plip** [comment]
oh -  I forgot about this one! Once we merge #10 ,  I think we could pretty safely copy this into the `install.service.sh` file:

```
sudo apt install -y python3-flask git screen python3-picamera2 python3-dotenv curl
cd ~
git clone https://github.com/NegativeK/picamserver.git
cd picamserver
mkdir data listeners
```

Then I think you should be able just run this one liner, yeah?

```
wget -O - https://raw.githubusercontent.com/NegativeK/picamserver/refs/heads/develop/install.service.sh  | sudo bash
```

(maybe use `main` instead of `develop`)

Happy to take a crack at this after we merge the other stuff!
